lib/pages/create_list.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:MobileOne/localization/localization.dart';
+import 'package:MobileOne/services/user_service.dart';
+import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+import 'package:uuid/uuid.dart';
+
+const Color ORANGE = Colors.deepOrange;
+const Color TRANSPARENT = Colors.transparent;
+
+int itemCounts = 0;
+
+class CreateList extends StatefulWidget {
+  State<StatefulWidget> createState() {
+    return CreateListPage();
+  }
+}
+
+class CreateListPage extends State<CreateList> {
+  final databaseReference = Firestore.instance;
+  final _myController = TextEditingController();
+
+  var _timeStamp = new DateTime.now();
+  var uuid = Uuid();
+
+  addListToDataBase() async {
+    var newUuid = uuid.v4();
+    bool doesListExist = false;
+
+    //Create a wishlist
+    await databaseReference.collection("wishlists").document(newUuid).setData({
+      'itemCounts': itemCounts.toString(),
```

Comment:
assumine thae you are creating a new wishlist it will be empty so you can specify "0" and remove int itemCounts = 0; at line 11